src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,390 @@
+---
+title: Organize CockroachDB Cloud Clusters Using Folders
+summary: Learn how to use folders to organize CockroachDB Cloud clusters and manage access to them.
+toc: true
+docs_area: manage
+---
+
+{% include_cached feature-phases/limited-access.md %}
+
+This page explains how to organize and manage access to your {{ site.data.products.db }} organization's clusters using folders. For more details about managing access to {{ site.data.products.db }} resources, refer to [Managing Users, Roles, and Service Accounts in {{ site.data.products.db }}](managing-access.html).
+
+## How folders work
+
+Folders provide additional ways to organize clusters and manage access to them. A folder can contain a mix of folders and clusters. Roles assigned on a folder are inherited by its descendants.
```

Comment:
```suggestion
Folders provide a way to organize clusters and manage access to them. A folder can contain a mix of folders and clusters. Roles assigned on a folder are inherited by its descendants.
```

Additional to what? If there's not another way of organizing clusters, use the suggested edit and remove.